src/libraries/System.Net.Http/src/System/Net/WebTransport/WebTransportException.cs

```diff
@@ -0,0 +1,11 @@
+using System.Runtime.Serialization;
+
+namespace System.Net.WebTransport;
+
+public class WebTransportException : Exception
+{
+    public WebTransportException(): base() { }
+    public WebTransportException(string message): base(message) { }
+    public WebTransportException(string message, Exception innterException): base(message, innerException) { }
```

Comment:
typo
```suggestion
    public WebTransportException(string message, Exception innerException): base(message, innerException) { }
```

Reply:
BTW QuicException doesn't have this one as public, only as internal, because only our code is supposed to throw this one, not the end-user, but it's just a nit.

Looking at QuicException and WebSocketException, and given that WT defines their error codes, I think it might make sense to add an enum and/or an integer corresponding to these error codes, so we could expose them to the end-user from the exception as a field, not only as a message.

NOTE: you can address this later